### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mbabker**
- B/C Break - Renames the format method to formatLocal - As this class extends DateTime, our format method declaration technically creates a strict standards error with the parent class so this flat out renames it.
- Adds Scrutinizer support
- Cleanup in the test class

### CODE REVIEW

**Comment by wilsonge** [comment]
If we need to have the formatLocal would it not be better the locale param. If you call format then it's effectively false and if you call formatL.ocal it's true...

**Comment by mbabker** [comment]
Umm, what?

[DateTime::format()](http://php.net/manual/en/datetime.format.php) does not have a locale parameter.  Hence the strict standards issue (even if one isn't thrown by the code).

**Comment by wilsonge** [comment]
So I'm saying effectively remove the local param so if you had

`$date->format($format, false)` you'd just call `$date->format($format)`

and if you had

`$date->format($format, true)` you'd just call `$date->formatLocal($format)`

**Comment by mbabker** [comment]
Ahh, much clearer than your first post :smiley:

**Comment by wilsonge** [comment]
My bad!!

**Comment by mbabker** [comment]
Updated

File: src/Date.php [hunk 1]
```diff
@@ -122,51 +121,51 @@ public function __get($name)
 		switch ($name)
 		{
 			case 'daysinmonth':
-				$value = $this->format('t', true);
```
Comment (wilsonge) [inline src/Date.php:125]: Is this b/c break here going from `locale=true` true to `locale=false` deliberate?

Comment (mbabker) [inline src/Date.php:125]: Yes.  The second param in the old format method basically decided whether to convert the time to GMT or not (hence me renaming it from `formatLocal` to `formatGmt`).  If false, it was converted.

**Pull request merged**